### PR TITLE
Adding agent/poller installation support for RPC-R

### PR DESCRIPTION
**Pull request by tonytan4ever**

### CODE REVIEW

**Comment by briancurtin** [comment]
recheck_all

**Comment by tonytan4ever** [comment]
recheck_xenial_ceph_deploy

**Comment by tonytan4ever** [comment]
recheck_xenial_newton_deploy

File: playbooks/maas-agent-setup.yml [hunk 1]
```diff
@@ -41,9 +42,9 @@
       file:
         path: "{{ maas_plugin_dir }}"
         state: directory
-        owner: "root"
+        owner: "{{ ansible_ssh_user | default('root')}}"
         group: "root"
-        mode: "0755"
+        mode: "0757"
```
Comment (npawelek) [inline playbooks/maas-agent-setup.yml]: Is there a reason this was changed from `0755` to `0757`? We shouldn't want these writeable by everyone.

Comment (tonytan4ever) [reply playbooks/maas-agent-setup.yml]: This is what's there left in the feature branch:
https://github.com/rcbops/rpc-maas/pull/548/files#diff-91e2993f1af4f36c72349d76a45c789dR152

I vaguely recall it's that if we don't use 757,  we will run into a permission issue ?

Comment (tonytan4ever) [reply playbooks/maas-agent-setup.yml]: Changed back to 755

**Pull request merged**